src/Fantomas.Core/SourceParser.fs

```diff
@@ -1071,7 +1071,8 @@ let (|Record|_|) =
 
 let (|AnonRecord|_|) =
     function
-    | SynExpr.AnonRecd (isStruct, copyInfo, fields, _) -> Some(isStruct, fields, Option.map fst copyInfo)
+    | SynExpr.AnonRecd (isStruct, copyInfo, fields, StartEndRange 1 (openingBrace, _, closingBrace)) ->
```

Comment:
I think it should be `StartEndRange 2` here, as the size you pass to the active pattern is the character length of the token. For `{|` and `|}` that would be 2.